src/auth-service/controllers/create-user.js

```diff
@@ -583,6 +701,20 @@ const createUser = {
         );
       }
 
+      if (!isEmpty(tenant) && tenant != "airqo") {
+        logObject("tenant", tenant);
```

Comment:
for cases where the tenant is not AirQo, do not allow login operation. 
One catch is that if the tenant is not provided, it will always default to AirQo. 
cc: @BenjaminSsempala @123MwanjeMike